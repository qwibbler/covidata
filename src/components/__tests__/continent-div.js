import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import ContinentDiv from '../continent-div';

it('renders correctly', () => {
  const component = renderer.create(
    <Router>
      <ContinentDiv country={{ name: 'Pakistan', href: 'http://PK/' }} population={100} confirmed={9} />
    </Router>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
