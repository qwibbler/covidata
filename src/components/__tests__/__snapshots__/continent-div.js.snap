// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className=" tab"
  href="#/continent/undefined/undefined/Pakistan/PK/"
  onClick={[Function]}
>
  <div
    className="country tab"
  >
    <div
      className="div-bg"
      style={
        Object {
          "opacity": 0.38,
        }
      }
    />
    <h2>
      Pakistan
    </h2>
    <div
      className="info block"
    >
      <h3>
        100
      </h3>
      <p>
        Population
      </p>
      <h3>
        9
      </h3>
      <p>
        Confirmed
      </p>
      <h3>
        0.09
        %
      </h3>
      <p>
        Percent
      </p>
    </div>
  </div>
</a>
`;
